pom.xml

```diff
@@ -655,6 +655,27 @@
                     <artifactId>helidon-maven-plugin</artifactId>
                     <version>${version.plugin.helidon-build-tools}</version>
                 </plugin>
+                <plugin>
+                    <groupId>io.helidon.build-tools</groupId>
+                    <artifactId>helidon-enforcer-plugin</artifactId>
+                    <version>${version.plugin.helidon-build-tools}</version>
+                </plugin>
+                <plugin>
+                    <groupId>io.helidon.build-tools</groupId>
+                    <artifactId>helidon-services-plugin</artifactId>
+                    <version>${version.plugin.helidon-build-tools}</version>
+                    <configuration>
+                        <mode>fail</mode>
+                    </configuration>
+                    <executions>
```

Comment:
Can you split this with the plugin section, add just version management and configuration under pluginManagement and add the execution in the plugin section, also state the phase so that we can see it explicitly when reading the pom.

Reply:
Done